### PR TITLE
docs: cookies

### PR DESCRIPTION
**Pull request by Meemaw**
Add some documentation for cookies.

Note: this PR is targeting `support-mocking-cookies` and not `master`.

### CODE REVIEW

**Comment by toomuchdesign** [comment]
Good job, @Meemaw. Thanks for also taking care of the dark side of open source! 😄 

Hey @all-contributors, @Meemaw deserved the documentation badge!

**Review by toomuchdesign — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -91,6 +108,7 @@ A temporary workaround consists of [mocking global `console.error` to ignore the
 
 - Consider reusing Next.js code parts (not only types)
 - Consider supporting Next.js `trailingSlash` option
+- Don't provide `getInitialProps` with `ctx` object when called client-side
```
Comment (Meemaw) [inline README.md:111]: This can be removed 🙂 See https://github.com/toomuchdesign/next-page-tester/pull/45#issuecomment-739793391

**Comment by toomuchdesign** [comment]
Ok, our bot is not very flexible :(

@all-contributors please add @Meemaw for documentation

File: README.md (hunk 1 above)
Comment (toomuchdesign) [reply README.md:111]: 🤟

**Pull request merged**